src/components/producerItems/actions.js

```diff
@@ -30,6 +33,20 @@ export const requestProductUpdate = payload => ({
     type: REQUEST_PRODUCT_UPDATE,
 });
 
+/**
+ * Triggers request to update product item details in the database
```

Comment:
update or delete?

Reply:
The action type is REQUEST_PRODUCT_DELETE, after the item is successfully deleted, it send back a success message which i created SUCCESS_MESSAGE, ERROR_MESSAGE actions to handle

Reply:
Good, update the comment to denote what it does

Reply:
ok.